.github/ISSUE_TEMPLATE/new_package.yml

```diff
@@ -0,0 +1,75 @@
+name: 🆕 New Package
+description: Propose the addition of a new package/tool
+title: <package_name>
+labels: [":new: package"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for contributing to [VM-Packages](https://github.com/mandiant/VM-Packages), your package proposal helps having more available tools for [FLARE-VM](https://github.com/mandiant/flare-vm)! :cupid: Please ensure that the tool doesn't already exist as part of any of the [current packages](https://github.com/mandiant/VM-Packages/tree/main/packages) and that there is no [issue](https://github.com/mandiant/VM-Packages/issues?q=is%3Aopen+is%3Aissue+label%3A%22%3Anew%3A+package%22) already.
+
+        ### Use the package name as the issue title.
+        The convention is to use lowercase names with the following format: `toolname.vm` or `tool-name.vm`. For example `capa.vm` and `fakenet-ng.vm`.
+  - type: input
+    id: name
+    attributes:
+      label: Tool Name
+      description: The name of the tool being installed, normally different from the package name. For example fakenet-ng.vm (package name) vs FakeNet-NG (tool name).
+      placeholder: ex. FakeNet-NG
+    validations:
+      required: true
+  - type: dropdown
+    id: browsers
+    validations:
+      required: true
+    attributes:
+      label: Category
+      description: Which category should this tool be installed to?
+      options:
+        - Android
+        - Debuggers
+        - Delphi
+        - Developer Tools
+        - Disassemblers
+        - dotNet
+        - Flash
+        - Forensic
+        - Hex Editors
+        - Java
+        - Javascript
+        - Net
+        - Office
+        - PDF
+        - Pentest
+        - PowerShell
+        - Python
+        - Text Editors
+        - Utilities
+        - VB
+        - Web Application
+  - type: input
+    id: url
+    attributes:
+      label: Download URL
```

Comment:
meta packages will require a dependency instead, should users input the link to chocolatey in that case here?

Reply:
We may want to have an option for "Is metapackage?" with links to supporting documentation for what a metapackage is: https://madvantage.atlassian.net/wiki/spaces/FLAR/pages/11625349790/NSIS+Installers

If it is a metapackage, then ask the user to provide the Chocolatey package name and locked version

Reply:
@MalwareMechanic you pasted the wrong link, but I know where to find the documentation. 😉 

I think we need to split packages types. Should we use this template for the default case and then we can have PRs for every specific type together with the automation? 

If this is then the default type (the one that is not going to be used for automization) I think we should remove the Download hash and make the download URL optional. Would that be ok @MalwareMechanic ?